### PR TITLE
Update kbo service

### PR DESCRIPTION
**Pull request by aliokan**
- Refactor to improve readability
- Add tests to improve reliability
- Replace `lblodgeneriek:geplandeEindDatum` by `dcterms:modified` to store `changeTime`
- Fix: Add missing `worship-service` graph in `update` contact, address and kboOrganisation
- Fix: email/website fields ID value
- Fix update queries (Update was failing when one property is missing)
- Fix `getAllAbbKboOrganizations` (All kboOrg properties were mandatory)
- Update README

### CODE REVIEW

File: eslint.config.js
Comment (Dachamelemon) [inline eslint.config.js:1]: **QUESTION:** Just for information mainly, what do you configure in this file exactly?

File: lib/wegwijs-api.js [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+  ACTIVE: "http://lblod.data.gift/concepts/63cc561de9188d64ba5840a42ae8f0d6",
+  INACTIVE: "http://lblod.data.gift/concepts/d02c4e12bf88d2fdf5123b07f29c9311",
+};
+
+/**
+ * Represents a Validty
+ * @typedef {object} Validity
+ * @property {string} [start] - The start date of the validity
+ * @property {string} [end] - The end date of the validity (optional)
+ */
```
Comment (Dachamelemon) [inline lib/wegwijs-api.js:13]: **Note:** I like this way of creating objects. Good to know!

File: .eslintrc.js
Comment (Dachamelemon) [inline .eslintrc.js:1]: **Question:** I assume you replaced this file by this one: https://github.com/lblod/sync-wegwijs-organization-service/pull/5/files#diff-a32a0887ed9d1d707bbb3b845b7df7fd40e673c47e7b60a3ebd896b68d3b8839 ?

**Review by Dachamelemon — APPROVED** [review]
Looks great! 

I just asked a few questions to help me understand some configurations you did, which i mainly related to woodpecker I think.

File: eslint.config.js
Comment (aliokan) [reply eslint.config.js:1]: It's the config file of [eslint](https://eslint.org/)

File: .eslintrc.js
Comment (aliokan) [reply .eslintrc.js:1]: Yes, `.eslintrc.js` is the old config format https://eslint.org/docs/latest/use/configure/migration-guide

**Pull request merged**